projects/plugins/crm/admin/contact/view.page.php

```diff
@@ -890,7 +890,7 @@ function jpcrm_render_contact_view_page( $id = -1 ) {
 
 								// if we have more than we're showing, communicate that
 								// Note this is defunct until we add contact filters to our object list views.
-								if ( $contact_invoice_count_inc_deleted > $contact_invoice_count ) {
+								if ( count( $contact['invoices'] ) > $contact_invoice_count_inc_deleted ) {
```

Comment:
I'm pretty sure by definition these are already equal as written. :^)

Reply:
Yes, I believe that's currently the intention based just on the comment above the if statement, also based on how it was written before 😄 
I did think about chopping the whole if statement (or rather just checking the variable exists) but it seems like there may be future plans for adding a filter, in which case checking the manual count with the variable that currently holds the count would apparently be the way that's used to check if something else is being filtered... either way I thought it best to leave the same kind of logic still in place for now.

Reply:
Yep, that's fine! Just figured I'd note it.